src/shims/posix/linux/foreign_items.rs

```diff
@@ -192,3 +201,61 @@ fn getrandom<'tcx>(
     this.write_scalar(Scalar::from_machine_usize(len, this), dest)?;
     Ok(())
 }
+
+fn futex<'tcx>(
```

Comment:
Could you move this function to a different file? `linux/sync.rs` seems appropriate.

We try to keep actual logic out if this dispatch file.

Reply:
Done